packages/core/src/_test/constants.ts

```diff
@@ -996,7 +997,7 @@ export const blockOneTransactions: RpcTransaction[] = [
 
 export const blockOneLogs: RpcLog[] = [
   {
-    address: "0x15d4c048f83bd7e37d49ea4c83a07267ec4203da",
+    address: "0x15D4c048F83bd7e37d49eA4C83a07267Ec4203dA",
```

Comment:
Was this necessary? My expectation is that now, it doesn't matter whether RPC responses are lowercase or checksummed.

Reply:
In some cases we are using this value as the return from the rpc and in some cases we are using it as the return from the db. In the return from db case, the value is always checksummed because we checksum the value before we add it to the db. I changed this back and explicitly checksummed in the few cases when we expected this value to be checksummed.